RELEASE_NOTES.md

```diff
@@ -12,6 +12,7 @@
 
 * Make a distinction between Order and Trade in the protobuf definitions
 * Introduction of new endpoints to retrieve gridpool trades
+* Addition of these new classes and methods to the Python client
```

Comment:
This note is a bit vague in the sense it doesn't tell the user what classes or methods have been added. The idea of the release-notes is to briefly describe and communicate, in this case, the new feature added without the need for the user to jump into the code to understand what the feature is about